EXPTracker.cs

```diff
@@ -359,6 +374,8 @@ public string ParseInput(string Text)
                     _host.SendText(@"#echo """"    """" Used to resume Exp Tracker");
                     _host.SendText(@"#echo /track report");
                     _host.SendText(@"#echo """"    """" Produce a report ");
+                    _host.SendText(@"#echo /track lowest {skills}");
+                    _host.SendText(@"#echo """"    """" Will #PARSE the lowest learning skill in the list.");

```

Comment:
That's actually (unintentionally) hidden in the "/track lowest" help info when the user sends /track lowest with nothing else (line 346).
`_host.EchoText("[/track lowest] will return the name of the skill with the lowest learning rate in a pipe or space delimited list");`

I'm looking now to see if there's a way to update that on Github or if I need to fix it locally and push the changes again. Still learning Github.

Reply:
Yeah, the easiest way is to make a change in your local repo, commit to the local, and push the commit to your remote.  Because the PR is linked to your repo, it'll automatically update.